hilti/runtime/include/configuration.h

```diff
@@ -18,7 +18,7 @@ struct Configuration {
     Configuration();
 
     /** Stack size for fibers with individual stacks. */
-    size_t fiber_individual_stack_size = static_cast<size_t>(1 * 1024 * 1024);
+    size_t fiber_individual_stack_size = static_cast<size_t>(256 * 1024 * 1024);
```

Comment:
On platforms with overcommitment completely disabled this can cause serious memory issues now, especially given the increased size. I wonder whether we should (e.g., on HILTI rt init) check whether overcommitment is enabled, and if not either fall back to shared stacks, or emit a warning.

Reply:
I reverted back to a size of 1MB since e.g., even Cirrus seems to disable memory overcommittment (that's at least how I interpreted the test suite hanging on practically all machines). Back at 1MB it now seems to pass again.

I pull fiber stats from a large internal Zeek analyzer we have at Corelight, and it looks like that analyzer had at most a less than 100 fibers active at a time, so individual stacks cost us worst case 100MB there as well (though in reality: much less due to overcommittment).